### ISSUE TITLE
Add back color settings

### ISSUE THREAD

**Issue by fedarko** [marbl/MetagenomeScope#367]
In old versions of the tool I set up a dialog that let users change any of the colors involved in the visualization:

<img width="983" height="776" alt="Image" src="https://github.com/user-attachments/assets/2f41cf4e-1066-4dc3-ae59-ee537c175ca6" />

We should add this back in some form. The main benefit IMO is adjusting patterns - this would enable something like using various shades of gray to represent patterns feasible, which would help make that side of things accessible. (Maybe doing this in a way that makes it easy to perfectly distinguish all five types of patterns is not easy, but it would def be possible to e.g. use dark gray for bubbles and light gray for frayed ropes etc)

**Comment by fedarko** [comment]
What library should we use to let the user select colors? (I mean the lazy thing is just having them type in a hex code but obviously that is insufficient.)

In the olden days, we used bootstrap colorpicker (https://farbelous.github.io/bootstrap-colorpicker/ -- the url is down but i think this corresponds to https://github.com/itsjavi/bootstrap-colorpicker) and it was perfect.

Nowadays, we have many options for what library to use. See https://community.plotly.com/t/proper-popup-colorpicker-in-dash/70077 for some dash-related discussion. I agree with the person there that, of the discussed options, the Bootstrap interface for this is the best (see https://www.dash-bootstrap-components.com/docs/components/input/ under section "Color picker"). Although it is still a little inelegant -- because this is really using https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/input/color, this falls back to the system-level colorpicker, and at least on xubuntu I am not a huge fan of the system colorpicker interface.

<img width="720" height="542" alt="Image" src="https://github.com/user-attachments/assets/19c0cbf1-fe3a-430e-bb15-305fa748681d" />

Seems like many fancy JS ones now are react modules that you would have to import or something (e.g. https://casesandberg.github.io/react-color/, https://github.com/omgovich/react-colorful). But there are many other options:

- https://jscolor.com/
- https://github.com/web-padawan/vanilla-colorful
- https://vanilla-picker.js.org/
- https://iro.js.org/
- https://simonwep.github.io/pickr/
- https://github.com/sefianecho/alwan (seems good for accessibility?)
- https://coloris.js.org/ (also seems accessible)

One consideration that comes up a lot here is accessibility (e.g. https://github.com/Myndex/SAPC-APCA/issues/143, https://github.com/simonwep/pickr/issues/11). The default [`<input type="color">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/input/color) thing is apparently sufficiently accessible, but it seems like many other JS solutions do not necessarily meet this criteria (e.g. assuming that the user will have a mouse).

So I am leaning towards just using the default color inputs -- seems simplest to integrate with dash, and least likely to introduce weird problems. Fitting it into an AG Grid table might be tricky, but we could always just use a custom HTML table for that...

I tried out the `<input type="color">` on my android phone and was surprised to see that [Firefox for Android only allows selection from a tiny list of colors](https://reddit.com/r/firefox/comments/ym0itq/i_love_firefox_but_the_firefox_android_color) -- Chrome at least lets you get to a HSV selector. If the desktop browsers guarantee that the user can at least pick any RGB color (and are not constrained to a silly list) then i will feel better about going with the default `<input type="color">` thing...
